tests/integration/common/helpers/test_collections.py

```diff
@@ -243,3 +245,30 @@ def test_current_question_does_not_exist(self, db_session, factories):
                 r"Could not find form for question_id=00000000-0000-0000-0000-000000000009 "
                 r"in collection_schema=[a-z0-9-]+"
             )
+
+    class TestGetAndSubmitAnswerForQuestion:
+        def test_submit_valid_data(self, db_session, factories):
+            section = factories.section.build()
+            form = factories.form.build(section=section)
+            question = factories.question.build(form=form)
```

Comment:
```suggestion
            question = factories.question.build()
```

The question will walk itself back up to form/section/schema for us - so can cut a bit of cruft here if you want.

Reply:
Thanks I did feel like I was overkilling the scenario but just ended up copying and pasting as it built - will make this change just so that doesn't propagate!

Reply:
Thanks this was helpful have made the change https://github.com/communitiesuk/funding-service/pull/216/commits/2de422a82fe1bc2762a78ec5bb5288973be08311